otherlibs/unix/unix.mli

```diff
@@ -918,33 +918,37 @@ val open_process_full :
    {!open_process_args_full} can be used instead of
    {!open_process_full}. *)
 
-val open_process_args_in : string -> string array -> in_channel
-(** [open_process_args_in prog args] runs the program [prog] with arguments
-    [args].  The new process executes concurrently with the current process.
-    The standard output of the new process is redirected to a pipe, which can be
-    read via the returned input channel.
-
-    The executable file [prog] is searched in the path. This behaviour changed
-    in 4.12; previously [prog] was looked up only in the current directory.
+val open_process_args : string -> string array -> in_channel * out_channel
+(** [open_process_args prog args] runs the program [prog] with arguments
+    [args].  Note that the first argument is by convention the name of the
+    file associated with the program being executed, just like
```

Comment:
Nit: could be more succint:
```suggestion
    [args].  Note that the first argument is by convention the name of the
    program being executed, just like
```

Reply:
Thanks for the review. I don't think this suggestion is an improvement, however. I was trying to echo `execve(2)`, which says:

> By convention, the first of these strings (i.e., argv[0]) should contain the filename associated with the file being executed.

The key point here is that it's the filename, not the name of the program.